### PR TITLE
Updating post

### PR DESCRIPTION
**Pull request by yadlra**

### CODE REVIEW

**Review by ocripps24 — APPROVED** [review]
Gorgeous!

File: config/routes.rb [hunk 1]
```diff
@@ -5,6 +5,8 @@
   get '/login', to: 'sessions#new'
   post '/login', to: 'sessions#create'
   delete '/logout', to: 'sessions#destroy'
+  get '/edit', to: 'posts#edit'
```
Comment (shektor) [inline config/routes.rb:8]: Should be covered by resource for posts, can be deleted

File: app/views/posts/edit.html.erb [hunk 2]
```diff
@@ -0,0 +1,29 @@
+<h1>Update post</h1>
+
+<%= form_with(model: @post, local: true) do |form| %>
+
+<% if @post.errors.any? %>
+<div id="error_explanation">
+  <h2>
+    <%= pluralize(@post.errors.count, "error") %>
```
Comment (shektor) [inline app/views/posts/edit.html.erb:8]: Can we have a short meeting to understand what this part of the code is doing.

**Review by shektor — APPROVED** [review]
Great testing of the new update action, may just need to add a feature test.

**Pull request merged**